src/templates/base.html

```diff
@@ -88,7 +88,7 @@
   <hr>
   <div class="container">
     <footer>
-      © CubingUSA 2017.
+      © CubingUSA {{ c.currentDate.year }}.
```

Comment:
should now be current_date

Reply:
Ah I knew I should have run it locally to make sure, should be fixed now